bin/cml/runner.js

```diff
@@ -436,6 +439,12 @@ exports.handler = async (opts) => {
 exports.builder = (yargs) =>
   yargs.env('CML_RUNNER').options(
     kebabcaseKeys({
+      tpiVersion: {
```

Comment:
should this be `hidden: true`?

Reply:
Good point! 🤦🏼‍♂️ `cmlVersion` that is slightly less arcane is also hidden, so it makes sense ho hide both.

Reply:
* https://github.com/iterative/cml/pull/967